libwebrtc/src/native/audio_source.rs

```diff
@@ -78,8 +78,10 @@ impl NativeAudioSource {
                     interval.tick().await;
 
                     let inner = source.inner.lock().await;
-                    if inner.captured_frames > 0 {
-                        break; // User captured something, stop injecting silence
+                    if let Some(last_capture) = inner.last_capture {
+                        if last_capture.elapsed() < Duration::from_millis(20) {
+                            break;
```

Comment:
should this be continue? for the next run?

Reply:
right, this is a mistake, will fix now